Docs/source/usage/parameters.rst

```diff
@@ -86,21 +86,22 @@ Overall simulation parameters
 
```

Comment:
Maybe we should point out that `evolve_scheme` is only relevant when using a Maxwell solver. (It is currently ignored when using WarpX with an electrostatic solver or with the Ohm's law hybrid solver.)